native/src/modules/categories/components/CategoryListContent.js

```diff
@@ -0,0 +1,181 @@
+// @flow
+
+import React from 'react'
+import { Dimensions, Text, View } from 'react-native'
+import { type DisplayMetrics } from 'react-native/Libraries/Utilities/NativeDeviceInfo'
+import Html, { GestureResponderEvent, type HTMLNode, type RendererFunction } from 'react-native-render-html'
+import { _constructStyles } from 'react-native-render-html/src/HTMLStyles'
+import styled from 'styled-components/native'
+import { type StyledComponent } from 'styled-components'
+import type { NavigationStackProp } from 'react-navigation-stack'
+import type { NavigateToInternalLinkParamsType } from '../../app/createNavigateToInternalLink'
+import type { ThemeType } from '../../../../build-configs/ThemeType'
+import { RTL_LANGUAGES } from '../../i18n/constants'
+import onLinkPress from '../../common/onLinkPress'
+
+const VerticalPadding: StyledComponent<{}, {}, *> = styled.View`
+  padding: 0 20px;
+`
+
+// type inferred from 'react-native/Libraries/Utilities/Dimensions'
+type DimensionsEventType = {
+  window?: DisplayMetrics,
+  screen?: DisplayMetrics,
+  ...
+}
+
+type ContentPropsType = {|
+  content: string,
+  navigation: NavigationStackProp<*>,
+  resourceCacheUrl: string,
+  navigateToInternalLink?: NavigateToInternalLinkParamsType => void,
+  cacheDictionary: { [string]: string },
+  language: string,
+  theme: ThemeType
+|}
+
+class CategoryListContent extends React.Component<ContentPropsType, {| width: number |}> {
+  constructor () {
+    super()
+    this.state = {
+      width: Dimensions.get('window').width
+    }
+  }
+
+  onChange = (dimensionsEvent: DimensionsEventType) => {
+    const defaultWidth = 200
```

Comment:
What is the effect of this defaultWidth? Is there a case when window is not defined? If yes we maybe just can skip this onChange event instead.